inference-engine/src/transformations/src/transformations/depth_to_space_fusion.cpp

```diff
@@ -91,10 +91,6 @@ void ngraph::pass::DepthToSpaceFusion::depth_to_space_fusion() {
     auto reshape_after = std::make_shared<ngraph::opset3::Reshape> (permute, input3, false);
 
     ngraph::graph_rewrite_callback callback = [this](pattern::Matcher& m) {
-        if (!transformation_callback(std::make_shared<ngraph::opset3::DepthToSpace>())) {
```

Comment:
@GlebKazantaev @itikhono Guys, now we have the same transformation_callback check in both DepthToSpace fusion and DepthToSpace decompose transformations. So from the plug-in I cannot disable only decomposition.  What's the proper way to handle this?

Reply:
current behavior: after enabling this callback you turn decomposition transformation off and turn fusion transformation on. Is it suitable for you now? I will make it possible to enable/disable each transformation independently a bit later.

Reply:
In fact there were 2 problems.
1) CommonOptimizations pass always used default transformations callback, so I wasn't able to specify the custom one.
2) In plug-in I uses the information regarding input rank of DepthToSpace layer (see mkldnn_plugin.cpp). That's why I moved callback call to the end of the fusion transformation to pass resulting DepthToSpace op in it.

Reply:
@GlebKazantaev @itikhono I need your approval regarding nGraph changes.